avahi-core/addr-util.h

```diff
@@ -42,6 +43,13 @@ int avahi_address_is_ipv4_in_ipv6(const AvahiAddress *a);
  * returns 1 if yes, 0 otherwise */
 int avahi_address_is_link_local(const AvahiAddress *a);
 
+typedef struct AvahiAddressList AvahiAddressList;
+
+struct AvahiAddressList {
+    AvahiAddress address;
+    AVAHI_LLIST_FIELDS(AvahiAddressList, address);
```

Comment:
Should we add also ``int prefix;`` member, which would allow storing network lists? If not specified, default to 32 of AF_INET and 128 for AF_INET6. Would make parsing it a bit more complicated. But I expect it would be desired.